.buildkite/autogenerate_pipeline.py

```diff
@@ -60,7 +60,7 @@
 
 # This represents the version of the rust-vmm-container used
 # for running the tests.
-CONTAINER_VERSION = "v19"
+CONTAINER_VERSION = "v21"
```

Comment:
How come we skipped one version?

Reply:
V20 was createad a weak back when you merged this I guess:

https://github.com/rust-vmm/rust-vmm-container/pull/77